pkgs/by-name/al/alexandria/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  rustPlatform,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  webkitgtk,
+  openssl,
+  callPackage,
+  libclang,
+  wrapGAppsHook4,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "alexandria";
+  version = "0.12";
+
+  src = fetchFromGitHub {
+    owner = "btpf";
+    repo = "Alexandria";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-NX89Sg639dSwUcUGrpgmdcO4tXl2iszctiRDcBVLbUA=";
+    fetchSubmodules = true;
+  };
+
+  webui = callPackage ./webui.nix {
+    inherit
+      meta
+      pname
+      version
+      src
+      ;
+  };
```

Comment:
To make overriding easier, please also `inherit` it into `passthru` (which is the usual pattern for webuis).